docs/learn/benefits-lukso-standards.md

```diff
@@ -47,18 +50,62 @@ As shown from the table above, the [Universal Profile Browser Extension](/insta
+### 🎨 Customizable Metadata
+
+A Universal Profile can represent many personas or forms of identities:
+
+- a user profile (personal or professional)
+- a brand
+- a DAO or an organisation
+- an AI living on the internet
+- a service like a wallet recovery
+- a character in a video game (represented as an NFT with its own metadata)
+
+<div style={{textAlign: 'center', color: 'grey'}}>
+  <img
+    src={require('/img/learn/different-universal-profiles.png').default}
+    alt="`Examples of different types of Universal Profiles."
+    width="1200"
+  />
+<br/>
+<i>4 x different Universal Profiles used for different purposes (a brand, a DAO, a user's profile, a wallet recovery service).</i>
+<br /><br />
+</div>
+
+### ⛽️ Gas-Less transactions
+
+:::info Opting out of the relayer
+
+**Using the relayer and always gas less transactions is not mandatory!**
+
+Users can also choose to not rely on the [Transaction Relay Service](../standards/accounts/lsp15-transaction-relayer-api.md) and the pay the gas fees of the transaction themselves by funding the [controller](./concepts.md#controller) of their Universal Profile with native LYX.
+
+:::
+
+Universal Profiles enable gas-less transactions. The **relayer pay the gas fees and does not require the user to hold native tokens** to pay for the transaction fee. Users do not need to go through the whole cumbersome process of:
+
+1. downloading a wallet
+2. creating a new address
+3. registering to a crypto exchange or a service to buy crypto
+4. performing KYC
+5. buying native crypto by card or bank transfer
+6. _in some cases,_ requesting the bank to authorize the transaction to buy crypto (some user banks restrict from buying crypto online)
+7. finally start using dApps!
+
+This improves onboarding experiences in the following ways:
+
+- most new web3 users does not get their head around the concept of "gas" and having to pay fees every time they interact with a dApp (compared to interacting with other applications on the internet).
+- native and experienced web3 users need to acquire more native currency from various chains just to _"play around with dApps"_ and explore, increasing the costs even more for users present across many chains.
+- users can also bridge native currency from one chain to another (_e.g: having to bridge ethers from Ethereum to Base to use dApps on Base_), but this require giving up some funds on one chain as a result.
+- developers looking to build dApps on new chains neex to acquire test LYX from testnet faucets, but most of them now require to hold a minimum amount of native currency on the main network to avoid spam and over-usage of the faucet.
```

Comment:
Suggesting to have similar terms `mainnet` and `testnet`.

```suggestion
- developers looking to build dApps on new chains need to acquire test LYX from testnet faucets, but most of them now require to hold a minimum amount of native currency on the mainnet to avoid spam and over-usage of the faucet.
```